introduction.md

```diff
@@ -1,2 +1,4 @@
-Hello
-JavaScript
+* Git
+* JavaScript
+* Ruby
+* HTML/CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?